ckanext/oidc_pkce/views.py

```diff
@@ -29,8 +30,15 @@ def get_blueprints():
     return [bp]
 
 
+def _no_cache():
+    # Ensure auth pages are not public cached
+    # Alternative way, generate anon user from ckan.common import current_user
+    request.environ[u'__no_cache__'] = True
```

Comment:
Shouldn't this also forbid private cache?

Reply:
I can't override whats in upstream ckan, i've got another PR in the works that will correct that behaviour in the coming days.

Reply:
Our fork already allows you to set `__no_private_cache__`

Reply:
https://github.com/qld-gov-au/ckan/blob/qgov-master-2.11.2/ckan/views/__init__.py#L64 (nope)
https://github.com/qld-gov-au/ckan/blob/qgov-master-2.10.7/ckan/views/__init__.py#L66 (nope)

It soon will once i get test passing:
https://github.com/ckan/ckan/compare/master...duttonw:ckan:feature/cache_tags